website/docs/r/logic_app_integration_account_partner.html.markdown

```diff
@@ -0,0 +1,83 @@
+---
+subcategory: "Logic App"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_logic_app_integration_account_partner"
+description: |-
+  Manages a Logic App Integration Account Partner.
+---
+
+# azurerm_logic_app_integration_account_partner
+
+Manages a Logic App Integration Account Partner.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_logic_app_integration_account" "example" {
+  name                = "example-ia"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  sku_name            = "Standard"
+}
+
+resource "azurerm_logic_app_integration_account_partner" "example" {
+  name                     = "example-iap"
+  resource_group_name      = azurerm_resource_group.example.name
+  integration_account_name = azurerm_logic_app_integration_account.example.name
+
+  business_identity {
+    qualifier = "ZZ"
+    value     = "AA"
+  }
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this Logic App Integration Account Partner. Changing this forces a new Logic App Integration Account Partner to be created.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the Logic App Integration Account Partner should exist. Changing this forces a new Logic App Integration Account Partner to be created.
+
+* `integration_account_name` - (Required) The name of the Logic App Integration Account. Changing this forces a new Logic App Integration Account Partner to be created.
+
+* `business_identity` - (Optional) A `business_identity` block as documented below.
+
+* `metadata` - (Optional) A JSON mapping of any Metadata for this Logic App Integration Account Partner.
+
+---
+
+A `business_identity` block exports the following:
+
+* `qualifier` - (Required) The business identity qualifier for the Logic App Integration Account Partner.
```

Comment:
what is a business identity qualifier

Reply:
Updated. Found the description from https://docs.microsoft.com/en-us/azure/logic-apps/logic-apps-enterprise-integration-partners